database/Image.js

```diff
@@ -0,0 +1,27 @@
+/* eslint camelcase: 0 */  // --> OFF
+const mongoose = require('mongoose');
+const db = require('./index.js');
+mongoose.Promise = global.Promise;
+
+
+const imageSchema = new mongoose.Schema({
+  title: String,
+  location: String,
+  rating: Number,
+  saved: Boolean,
+  shared: {
+    facebook: Boolean,
+    twitter: Boolean,
+    email: Boolean
+  },
+  image: {
+    living_room: String,
+    bedroom: String, 
+    dining_room: String,
+    den: String
+  }
+});
+
+const Image = mongoose.model('Image', imageSchema);
+
+module.exports.Image = Image;
```

Comment:
@ykent4 AirBnB styleguide would have you put a new line at the end of this file.

Reply:
thanks! changed